Scripts/script-TaniumFindRunningProcesses.yml

```diff
@@ -166,3 +167,4 @@ dependson:
   - tn-result-data
   - tn-result-info
 timeout: 0s
+releaseNotes: Deprecated script
```

Comment:
please write the command that replace this script.
Deprecated script, use `command name` instead.